### ISSUE TITLE
pip3 install error

### ISSUE THREAD

**Issue by nicolasesnis** [AlexIoannides/py-package-template#13]
```
myshell:~/SE/nico$ pip3 install py-template-project

Collecting py-template-project
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 353, in run
    wb.build(autobuilding=True)
  File "/usr/lib/python3/dist-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/usr/lib/python3/dist-packages/pip/req/req_set.py", line 554, in _prepare_file
    require_hashes
  File "/usr/lib/python3/dist-packages/pip/req/req_install.py", line 278, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 465, in find_requirement
    all_candidates = self.find_all_candidates(req.name)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 423, in find_all_candidates
    for page in self._get_pages(url_locations, project_name):
  File "/usr/lib/python3/dist-packages/pip/index.py", line 568, in _get_pages
    page = self._get_page(location)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 683, in _get_page
    return HTMLPage.get_page(link, session=self.session)
  File "/usr/lib/python3/dist-packages/pip/index.py", line 795, in get_page
    resp.raise_for_status()
  File "/usr/share/python-wheels/requests-2.18.4-py2.py3-none-any.whl/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://pypi.org/simple/py-template-project/
```

**Comment by johann-petrak** [comment]
This seems to happen because the Pipfile defines `url = "https://pypi.org/simple"`

It appears that this project is broken and abandoned as it did not see any updates for 3 years.

**Comment by jfthuong** [comment]
You need to do `pip install py-package-template` as indicated on top of the Pypi page. This install sucessfully. The command to install is correct.

**Comment by johann-petrak** [comment]
Actually, the pypi page / README file DOES say "pip3 install py-template-project".

See https://github.com/AlexIoannides/py-package-template/blob/11f37953683e5f063e89ca2c567304428e9cb563/README.md

**Comment by jfthuong** [comment]
Only the top of the Pypi is correct. The rest is indeed incorrect (as the content of Pypi is based on the README). I have done a Pull Request to correct it :)

![image](https://user-images.githubusercontent.com/1401809/123772065-26d94c80-d8fe-11eb-958a-8b9d460485e4.png)

**Comment by jfthuong** [comment]
FYI. Duplicate with #8

**Comment by AlexIoannides** [comment]
Closed by #15
